Server/JokesOnYou.Web.Api/DTOs/JokeDto.cs

```diff
@@ -1,17 +1,14 @@
-﻿using JokesOnYou.Web.Api.Models;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System;
 
 namespace JokesOnYou.Web.Api.DTOs
 {
-    public class JokeReplyDto
+    public class JokeDto
     {
         public int Id { get; set; }
         public string Premise { get; set; }
         public string Punchline { get; set; }
         public string AuthorId { get; set; }
```

Comment:
I'd rather have this as a nested object (something like JokeAuthorDto). But when I remove the AuthorId from JokeDto, then it will not be available for me to retrieve the name in the second query...

I'm lost :)

Reply:
A joke Always has an Author and will always be used imo.
so this is good.